src/style.css

```
@@ -0,0 +1,38 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100px;
+  height: 15px;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-last-child(n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-last-child(n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-last-child(n + 2) {
+  background-image: url(images/star-active.svg);
+}
```

you can combine these rules into one, like this

```suggestion

.stars--3 .stars__star:nth-last-child(n + 3),
.stars--4 .stars__star:nth-last-child(n + 2) {
  background-image: url(images/star-active.svg);
}
```